scripts/tf-validate.sh

```diff
@@ -14,6 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# TODO: remove this after releasing TPF
+if git diff --quiet -- ./internal/config/advanced_cluster_v2_schema.go; then
+  V2_SCHEMA_DISABLED=true
+else
+  V2_SCHEMA_DISABLED=false
+fi
+
+if $V2_SCHEMA_DISABLED; then
+  echo "enabling Advanced Cluster V2 Schema"
+  make enable-advancedclustertpf
```

Comment:
```suggestion
  make enable-preview-provider-v2
```

can be already changed

Reply:
Thanks!